.github/workflows/build.yml

```diff
@@ -169,30 +169,4 @@ jobs:
            echo "error: $ERROR"
            exit 1
          fi
-         URL=$(jq -r '.previewURLs[0]' <<< $RESULT)
-         WORKSPACE=$(jq -r '.workspaceIDs[0]' <<< $RESULT)
-         echo "::set-output name=preview::$URL"
-         echo "::set-output name=workspace::$WORKSPACE"
       shell: bash
```

Comment:
nit: can we put this line up so it's not lost after the indented logic block?

Reply:
Makes sense. Done.